docs/pages/platform/upgrading/1.2.mdx

```diff
@@ -96,9 +96,9 @@ with `authorize`, is still supported but will eventually be deprecated.
 ## Access tokens [#access-tokens]
 
 Access tokens are the new recommended way to authenticate, because they’re easy
-to manage from your custom back end. They follow the analogy of a “hotel room
-key card”. Anyone that has a key card can enter any room that the card gives
-access to. It’s easy to give out these key cards right from your back end.
+to manage from your custom back end. They follow the analogy of a _hotel key
```

Comment:
If we change this copy here, we likely want to change it in the authentication tooltip as well